products/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db   import models
+from core.models import TimeStampedModel
+
+class Product(TimeStampedModel):
+    product_name       = models.CharField(max_length=500)
+    category           = models.ForeignKey('Category', on_delete=models.CASCADE)
+    country            = models.ForeignKey('Country', on_delete=models.CASCADE)
+    alcohol_percentage = models.DecimalField(max_digits=3, decimal_places=1)
+    price              = models.DecimalField(max_digits=10, decimal_places=3)
+    average_rating     = models.DecimalField(max_digits=2, decimal_places=1)
+    content            = models.CharField(max_length=1000)
+    food_category      = models.ManyToManyField('FoodPairing', through="ProductFoodPairing")
+    property           = models.JSONField()
+
+    class Meta:
+        db_table = 'products'
+
+class ImageUrl(TimeStampedModel):
```

Comment:
다양한 이미지 테이블이 필요할 수도 있습니다.
```suggestion
class ProductImage(TimeStampedModel):
```